metrics/utils.go

```diff
@@ -16,11 +16,39 @@ limitations under the License.
 
 package metrics
 
-import "strconv"
+import (
+	"strconv"
+
+	"go.opencensus.io/tag"
+)
 
 // ResponseCodeClass converts an HTTP response code to a string representing its response code class.
 // E.g., The response code class is "5xx" for response code 503.
 func ResponseCodeClass(responseCode int) string {
 	// Get the hundred digit of the response code and concatenate "xx".
 	return strconv.Itoa(responseCode/100) + "xx"
 }
+
+// MaybeInsertInt conditionally insert the tag when cond is true.
+func MaybeInsertInt(key tag.Key, value int, cond bool) tag.Mutator {
```

Comment:
`ConditionallyDoXYZ` feels less uncertain than `MaybeDoXYZ` to me.

Reply:
That's a fairly common terminology in functional programming (see the Maybe monad).. But I digress. `Maybe` is shorter :-)

Reply:
If shorter means better I can't really argue.